source/development_process/index.rst

```diff
@@ -0,0 +1,994 @@
+conservative development and release cycle of the Protein Geometry Database,
+where every major change and new release must be certified by scientists.
+
+For the What's Fresh and Working Waterfronts website the developer team chose
+to follow the agile methodology. The agile method promotes quick responses to
+change and customer collaboration, which made it an ideal fit for working with
+both the Sea Grant Foundation and the BSG on an integrated website and mobile
+application. The agile development methodology is defined in the `Agile
+Manifesto`_, which is required reading for
+all developers, should the team choose to follow this methodology. When
+following the agile methodology the developer team meets every month to
+discuss the state of the project and show their progress to the client. The Sea
+Grant projects also followed the test driven development model, which dictates
+that tests should be written before code. This ensures that all of the code is
+tested and works properly.
+
+The PGD project has a very different set of constraints. It is a large project
+which is being maintained, but little active development is being done. The PGD
+is a tool used by hundreds of scientists throughout the world, so it is very
+important that changes do not introduce bugs which may affect ongoing research.
+Every change to the PGD project must be approved by the scientists in charge of
+the project. It is difficult to write tests to ensure that new changes don't
+impact the science. Historically the PGD has had issues with upstream protein
+repositories and antiquated closed-source dependencies. Even importing new data
+can sometimes be a fraught process, but the team is developing a detailed
+process to minimize problems when upstream software or data changes.
+
+.. _Agile Manifesto: http://www.agilemanifesto.org/
+
+Tools The Team Uses
+-------------------
+At the OSL developers use a variety of open source tools. Developers are
+expected to have at least passing familiarity with all of these tools. New
+developers are encouraged to research any of these tools they have not heard
+of.
+
+Most OSL websites are written in Python using the Django web application
+framework. Websites are deployed using Chef configuration management on CentOS
+6.6. Most of our sites use the MySQL database, although some use PostGres.
+Most OSL websites have a Docker development environment set up to make
+development fast and easy. To learn how to use the Docker development
+environment please consult the project documentation and the OSL's `internal
+documentation`_.
+The OSL uses ``git`` for version control and keeps projects on
+GitHub_. The OSL uses the Github issue tracker.
+
+.. _internal documentation: https://docs.osuosl.org/development/docker-dev-environments.html
+.. _GitHub: https://github.com/osuosl
+.. _RedMine: https://code.osuosl.org
+
+How a Typical Project is Written
+================================
```

Comment:
At this point the document has passed the point of being singular, and should be split out, since the `===` heading is reserved for the title, and this section goes way more in-depth than expected by the original tone of the introduction.